Source/WebCore/page/Quirks.cpp

```diff
@@ -1974,5 +1974,15 @@ bool Quirks::shouldIgnoreContentObservationForClick(const Node& targetNode) cons
 
 #endif // PLATFORM(IOS_FAMILY)
 
+bool Quirks::needsMozillaFileTypeForDataTransfer() const
+{
+    if (!needsQuirks())
+        return false;
+
+    if (isDomain("live.com"_s))
```

Comment:
Do we really want this on all live.com? Not just outlook?

Reply:
Oh, good call — `outlook.live.com` is probably more appropriate…